src/containers/Articles.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+import fetchPersonalArticles from '../store/actions/articleActions';
+import Articles from '../components/Articles';
+
+const mapStateToProps = state => ({
+  personalArticles: state.articles,
+});
+
+const mapDispatchToProps = dispatch => ({
+  fetchPersonalArticles: () => {
+    dispatch(fetchPersonalArticles());
+  },
+});
+
+export default connect(mapStateToProps, mapDispatchToProps)(Articles);
```

Comment:
This export could have been done elsewhere. Why are you making it in another file?

Reply:
Should I move this to the Articles.jsx file? I was trying to differentiate between the component and its connected counterpart